lib/config.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
+
+
+let _       = require('lodash'),
+    path    = require('path');
+
+class Config {
+
+    constructor(pkg, options, basePath) {
+
+        this.options = options;
+        this.basePath = basePath;
+        this.cacheControlHeader = this.options.maxAge || process.env.CACHE_CONTROL || 'max-age=60';
+        this.customHeaders = this.options.customHeaders || [];
+        this.description = this.options.description || pkg.description;
+        this.env = this.options.host || process.env.ENVIRONMENT || process.env.NODE_ENV || 'development';
+        this.envProd = this.options.envProd || 'production';
+        this.port = this.options.port || process.env.PORT || 3000;
+        this.host = this.options.host || process.env.HOST || '0.0.0.0';
+        this.name = this.options.name || pkg.name;
+        this.surrogateControlHeader = this.options.surrogateCacheControl || process.env.SURROGATE_CACHE_CONTROL ||
+            'max-age=360, stale-while-revalidate=60, stale-if-error=86400';
```

Comment:
The environment vars should supercede the passed in options. This allows for us to make changes without a code change. Please update.